apps/api/test/reports.e2e-spec.ts

```diff
@@ -0,0 +1,79 @@
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import request from 'supertest';
+import { randomUUID } from 'crypto';
+import dayjs from 'dayjs';
+
+import { AppModule } from 'src/app.module';
+import { AuthGuard } from 'src/auth/auth.guard';
+import { canActivate } from './override-guard';
+import { validApplicantForReportOne } from './fixture/ien';
+
+describe('Report Controller (e2e)', () => {
+  let app: INestApplication;
+  let applicanIdOne: string;
+  let applicanIdTwo: string;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    })
+      .overrideGuard(AuthGuard)
+      .useValue({ canActivate })
+      .compile();
+
+    applicanIdOne = randomUUID();
+    applicanIdTwo = randomUUID();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  it('Add Applicant /ien (POST)', done => {
```

Comment:
`it` defines a test case. so, these two cases should be moved into the last one.